psi4/src/psi4/libmints/dimension.cc

```diff
@@ -37,25 +37,20 @@
 #include "psi4/libpsi4util/exception.h"
 
 namespace psi {
-
 Dimension::Dimension() : name_("(empty)") {}
-
-Dimension::Dimension(int n, const std::string& name) : name_(name), blocks_(n, 0) {}
-
+Dimension::Dimension(const size_t n, const std::string& name/* = ""*/) : name_(name), blocks_(n, 0) {}
```

Comment:
Here `const size_t` seems superfluous since n is copied and used only once. I'm in favor of using `const` as much as possible, especially to explicitly express that a variable should not be changed. But here we copy n and it's only used once.

Reply:
I have a habit of doing "const by default", but it is indeed making thing more verbose than necessary here.
Change done.